Regression_CF.R

```diff
@@ -0,0 +1,69 @@
+#setwd("C:/Users/frase/Documents/Rclass/Regression-repo")
+
+read.csv("iris.csv")
+
+#rename vectors
+Slength <- iris$Sepal.Length
+Plength <- iris$Petal.Length
+
+x <- Slength
+y <- Plength
+lengths.lm <- lm(y~x) 
+coef(lengths.lm)
+b <- coef(lengths.lm)
+int <- b[1]
+slope <- b[2]
+
+plotwithreg <- function(x=Slength, y=Plength, title="Sepal and Petal Morph", cxlab="Sepal Length", cylab="Petal Length", ccol=iris$Species) {
```

Comment:
Claire, I love what youʻre doing here but you havenʻt put int and slope as arguments to the function. Therefore it is not changing. Thatʻs why some of your plots look funny. It is relying on how int and slope are defined in the global environment, and it may not be totally predictable. This is an easy fix, please do it.